pkg/reconciler/instances/istio/actions/performer_test.go

```diff
@@ -465,7 +465,7 @@ func Test_DefaultIstioPerformer_ResetProxy(t *testing.T) {
 		proxyImageVersion := "1.2.0"
 
 		// when
-		err := wrapper.ResetProxy(ctx, kubeConfig, proxyImageVersion, log)
+		err := wrapper.ResetProxy(ctx, kubeConfig, proxyImageVersion, "",log)
```

Comment:
Maybe it would be more fitting if you created env such as `proxyImageVersion` and make it empty.